tests/DummyTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Psr\Log\Util\Tests;
+
+/**
+ * This class is internal and does not follow the BC promise.
+ *
+ * Do NOT use this class in any way.
+ *
+ * @internal
```

Comment:
What's the need in declaring it internal? If it's under `tests` it shouldn't get published. If we need to publish a test case, we should put it under `src`, and maybe reuse it from `tests` to "test" it out.

Reply:
Technically, `DummyTest` would IMHO be a stub, and as such I would put this under tests, but perhaps in a deeper namespace, if that's preferable.

The `@internal` annotation is from `psr/log`, and I do not know why it is needed there either. No objections to remove.

Reply:
- Annotation removed in 9de4c99.
- Moved to `Stub` in 2a66682.
- Altogether removed class in b3fc5b0.

If you disagree with some of these changes, please let me know and I will revert.